roles/neutron-data-network/tasks/main.yml

```diff
@@ -262,6 +278,24 @@
         - /etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini
         - /etc/neutron/neutron_lbaas.conf
 
+- name: install neutron lbaas service (rhel distro)
+  systemd_service:
```

Comment:
yes, package provided a systemd service unit, but the content of the service has some difference to ours. Taking neutron lbaas as example, we copied lbaas_agent.ini to /etc/neutron/services/loadbalancer/haproxy/, however rpm uses location of /etc/neutron/. We have to overwrite these configuration of the service unit. In addtion, service unit creatd by red hat contains -dist.conf files and I removed them by overwriting the service unit.